change-beta/@azure-communication-react-3ae22222-4a77-4173-80b3-ec236bef5e2f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add CTE support to adapter layer",
```

Comment:
Let's not use `CTE` in the changelog. Can this be more descriptive? `Add TeamsCallAdapter etc. to support composites powered by teams identity` or some such?